openverse-api/catalog/api/utils/waveform.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+import math
+import mimetypes
+import os
+import pathlib
+import shutil
+import subprocess
+
+import requests
+
+log = logging.getLogger(__name__)
```

Comment:
Should this have a config set?

Reply:
Django logging is already configured here:

https://github.com/WordPress/openverse-api/blob/16294e8a381a7ca8c5c38b9bd81670e15fec7622/openverse-api/catalog/settings.py#L245